cli/src/common_utils/utility_scripts.sh

```diff
@@ -133,3 +133,21 @@ function ensure_healthy_cluster {
 
     _linfo "*****************************************************************"
 }
+function set_lrserial_number {
+    local _serial_number=${1:-}
+
+    _linfo "Setting Lyve Rack serial number"
+
+! read -r -d '' _script << EOF
+    if [[ $(/usr/local/bin/provisioner --version > /dev/null) ]]; then
+        /usr/local/bin/provisioner pillar_set cluster/lr_serial_number $_serial_number
+    fi
+    echo $_serial_number > /opt/seagate/lr_serial_number
```

Comment:
add check to verify the serial number is 15 char long

Reply:
i have added the check during the parameter input for serial number